src/database/sql/bundles/import.sql

```diff
@@ -1,43 +1,58 @@
 -- upsertBundle
 INSERT INTO bundles (
-  id, root_transaction_id, format_id,
-  unbundle_filter_id, index_filter_id,
-  data_item_count, matched_data_item_count,
-  first_queued_at, last_queued_at,
-  first_skipped_at, last_skipped_at,
-  first_unbundled_at, last_unbundled_at,
-  first_fully_indexed_at, last_fully_indexed_at,
-  import_attempt_count, duplicated_data_item_count
+id, root_transaction_id, format_id,
```

Comment:
Nit: let's preserve the original indentation here.

Reply:
https://github.com/ar-io/ar-io-node/pull/289/commits/4af080a12cea9ab3e2a6800aeba30d6320aa957a